arithmetization/src/main/java/net/consensys/linea/zktracer/types/MemoryRange.java

```diff
@@ -73,21 +73,21 @@ public MemoryRange(long contextNumber) {
   public MemoryRange(final long contextNumber, final Range range, final Bytes rawData) {
     this.contextNumber = contextNumber;
     this.range = range;
-    this.rawData = rawData;
+    this.rawData = range.isEmpty() ? Bytes.EMPTY : rawData;
```

Comment:
this (and following) are just to make the constructor consistent: if the range isEmpty, then don't store the rawData

Reply:
Yes